neptune/exceptions.py

```diff
@@ -59,9 +86,23 @@ def __init__(self):
         super(NoChannelValue, self).__init__('No channel value provided.')
 
 
-class LibraryNotInstalled(NeptuneException):
+class NeptuneLibraryNotInstalledException(NeptuneException):
```

Comment:
Oook, so I checked and this actually is used [here](https://github.com/neptune-ai/neptune-contrib/blob/c0eb668d6e04f90b51172748e2aa6849c18cc29b/neptunecontrib/monitoring/keras.py#L18). Very weird dependency, but changing this name will break contrib. So we probably need a backward compatible rename, but also removing this dependency from contrib seems like a good idea.

Reply:
Dropped it from contrib see this PR https://github.com/neptune-ai/neptune-contrib/pull/107/files